about.html

```diff
@@ -140,26 +149,21 @@ <h3 class="featurette-heading">Silicon Village</h3>
           build a technology echosystem | build a technology market |
           something about economy?-->
           With this in mind, they created the Silicon Village project with
-          the goal of ...
-          by creating a technology infrastructure and nurturing
-          the community to support it.
+          the goal of recruiting developers from Guam by creating a technology
```

Comment:
we don't want to recruit developers, we want to grow them.
also, it's not necessarily devs, but also IT people